rocksdb_admin/admin_handler.cpp

```diff
@@ -483,6 +483,9 @@ AdminHandler::AdminHandler(
       LOG(INFO) << "Stopping DB deletioin thread ...";
     });
   }
+
+  int upload_MBps = 600; // set as fixed for now.
+  s3_conc_ = std::make_shared<common::S3Concurrent>("pinterest-jackson", upload_MBps); 
```

Comment:
we need to use the bucket passed by the gflag to support for pii cluster, which will use `pinterest-bristol`

Reply:
I don't see the gflag for bucket ... s3_bucket, s3_bucket_backup just used in tests.

Reply:
There is a gflag for which s3 bucket to use: https://github.com/pinterest/rocksplicator/blob/master/common/helix_client.cpp#L35. But, use the bucket specified in the request should be good.

Reply:
OK will just use bucket from the request.